web/.eslintignore

```diff
@@ -0,0 +1,7 @@
+# don't ever lint node_modules
+node_modules
+# don't lint build output (make sure it's set to your correct build folder name)
+dist
+public
+# don't lint nyc coverage output
+coverage
```

Comment:
Let's not to break traditions and append new line here

Reply:
Done